packages/strapi-plugin-content-manager/admin/src/components/TableRow/index.js

```diff
@@ -41,7 +41,7 @@ class TableRow extends React.Component {
       case 'integer':
       case 'biginteger':
       case 'decimal':
-        return value && !isEmpty(value.toString()) ? value.toString() : '-';
+        return !isNull(value) ? value.toString() : '-';
```

Comment:
Not sure, but I think this should also be done with Integer and Float values. Just taking a look at an example of project of an integer value that is set to zero:

![image](https://user-images.githubusercontent.com/8593673/46600708-e4151d80-ca9f-11e8-8a79-5f45511c2f58.png)


![image](https://user-images.githubusercontent.com/8593673/46600714-eb3c2b80-ca9f-11e8-8206-b47805032950.png)

Reply:
DB also shows the value of 0 instead of null:

![image](https://user-images.githubusercontent.com/8593673/46600767-132b8f00-caa0-11e8-95cc-a70f42e513b6.png)

Reply:
Nope this was on a 14.2 project, didn't test directly just assumed based on where that code is :p

Reply:
Ah I see how this works now @lauriejim should be good I was thinking bookshelf files... Need more coffee